### DISCUSSION TITLE
[Discord] Feature suggestions

### DISCUSSION THREAD

**Discussion by TetraTheta** [EssentialsX/Essentials#4333] [Ideas and Feature Suggestions] [answered]
Since EssentialsX Discord module is on public, I think these can be discussed in here.

EssentialsX Discord is free of clutterness than *famous plugin*. But I miss some kind of feature of *it*.

* Lack of acheivement announcement  
  When player acheived acheivement, *it* can relay the message to Discord. I think this is attractive feature of *it*.
* Relay `/say` to Discord
  I don't know whether *it* can do it, but it would be nice to get `/say` is relayed to Discord like this:  
  
  ```
  TetraTheta >> Hello world!
  [Server] >> Hello world!
  ```
  Since player cannot have `[`, `]` in their name, this can represent that the message is from server. (well, aside of nickname though)

**Comment by TetraTheta** [comment]
Well, about acheivement is already working on at https://github.com/EssentialsX/Essentials/pull/4313 so only thing remain would be 'relay `/say` to discord' thing...

**Comment by JRoy** [comment, marked as answer]
`/say` doesn't fire its own event and there is no way of distinguishing it.
